mm/memory.c

```diff
@@ -208,6 +212,11 @@ int free_page_tables(struct proc *p)
 }
 
 unsigned int map_page(struct proc *p, unsigned int vaddr, unsigned int addr, unsigned int prot)
+{
+	return map_page_flags(p, vaddr, addr, prot, 0);
+}
+
+unsigned int map_page_flags(struct proc *p, unsigned int vaddr, unsigned int addr, unsigned int prot, int flags)
```

Comment:
Where is the function `map_page_flags()` used?

Reply:
This is used in the framebuffer driver `fb_mmap` function, which is the where the PAGE_NOALLOC flag is passed. In this way, all the other calls to `map_page` a guaranteed to work as before, with only the framebuffer mmap operating differently.

Reply:
> You mean that this function will be used on a later PR?

No, the function `fb_mmap` requires and calls `map_page_flags` now in this PR :)

Reply:
Damn!; I'm blind. I see it now.
Sorry.